### ISSUE TITLE
new package: shiny app that leverages existing RECON packages

### ISSUE THREAD

**Issue by nistara** [RECON-gathering-March-2018/projects#5]
There are package specific shiny apps (e.g. the one for `epicontacts`), but it could be useful to create one main shiny app for all main tools and visualizations generated by existing RECON or other useful packages. Something to visualize (perhaps interactively) an outbreak and its epidemiological parameters, other analyses etc in a single place.

**Comment by thibautjombart** [comment]
Yep yep, that's been on my mind for a while, indeed! I think this would need to be modular, e.g.

```
shine_recon(incidence = TRUE, contacts = TRUE, earlyR = TRUE, ...) # each arg corresponds to a module (ie one or more tab)
```

What do you think?

**Comment by nistara** [comment]
I agree, something like that indeed!
